frontend/src/components/Asset.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+    function round(value: number): number {
+        return Math.round(100 * value) / 100;
+    }
+
+    interface Asset {
+        company: string;
+        symbol: string;
+        qty: number;
+        currentVal: number;
+        glToday: number;
+        glOverall: number;
+    }
+
+    export let data: Asset;
+
+    const colorToday = data.glToday >= 0 ? "success" : "warning";
+    const colorOverall = data.glOverall >= 0 ? "success" : "warning";
+</script>
+
+<div class="block">
+    <div class="columns mx-2 is-vcentered" style="margin: auto;">
+        <div class="column is-2 has-text-centered">
```

Comment:
What's the purpose of is-2 here?

Reply:
Column width 2 out of 12

Reply:
Huh, learnt something new. I’ve always just used the is-x-yths syntax 😅